src/containers/BooksList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+function BooksList({ bookList }) {
```

Comment:
## Convert to ES6 syntax
- [ ] We need to ensure we are using ES6 syntax all throughout the codebase. Kindly convert this `BookList` component function declaration to an arrow based function.
```suggestion
const BooksList = ({ bookList }) => {
```